### PR TITLE
fix some python3 incompatibilities

### PR DESCRIPTION
**Pull request by collinanderson**

### CODE REVIEW

File: src/libstd_unicode/unicode.py [hunk 1]
```diff
@@ -519,32 +519,29 @@ def emit_conversions_module(f, to_upper, to_lower, to_title):
...
 
-    compat_keys = compat.keys()
-    compat_keys.sort()
+    compat_keys = sorted(compat.keys())
 
     canon_comp = {}
     comp_exclusions = norm_props["Full_Composition_Exclusion"]
     for char in canon_keys:
-        if True in map(lambda (lo, hi): lo <= char <= hi, comp_exclusions):
+        if True in map(lambda lo_hi: lo_hi[0] <= char <= lo_hi[1], comp_exclusions):
```
Comment (kennytm) [inline src/libstd_unicode/unicode.py]: Maybe refactor this to?

```python
if any(lo <= char <= hi for (lo, hi) in comp_exclusions):
    continue
```

File: src/libstd_unicode/unicode.py [hunk 2]
```diff
@@ -446,14 +446,14 @@ def emit_bool_trie(f, name, t_data, is_pub=True):
     f.write("    };\n\n")
 
 def emit_small_bool_trie(f, name, t_data, is_pub=True):
-    last_chunk = max(int(hi / 64) for (lo, hi) in t_data)
+    last_chunk = max(int(hi // 64) for (lo, hi) in t_data)
```
Comment (kennytm) [inline src/libstd_unicode/unicode.py]: Will we ever get non-integer from the source code? All the `int()` calls could be dropped in not.

**Review by kennytm — APPROVED** [review]
LGTM. Just some nits.

**Comment by collinanderson** [comment]
Hah. Good points. Fixed both.

**Comment by kennytm** [comment]
Thanks!

@bors r+ rollup

**Pull request merged**